_integration/testsuite/install-contour-working.sh

```diff
@@ -64,13 +64,129 @@ for t in $TAGS ; do
     kind::cluster::load docker.io/projectcontour/contour:$t
 done
 
+
 # Install Contour
-#
+
+${KUBECTL} apply -f ${REPO}/examples/contour/00-common.yaml
+
+# Update the config file to set a GatewayAPI Gateway.
+${KUBECTL} apply -f ${REPO}/examples/contour/01-crds.yaml
+${KUBECTL} apply -f ${REPO}/examples/contour/02-rbac.yaml
+${KUBECTL} apply -f ${REPO}/examples/contour/02-role-contour.yaml
+${KUBECTL} apply -f ${REPO}/examples/contour/02-service-contour.yaml
+${KUBECTL} apply -f ${REPO}/examples/contour/02-service-envoy.yaml
+
 # Manifests use the "Always" image pull policy, which forces the kubelet to re-fetch from
 # DockerHub, which is why we have to update policy to `IfNotPresent`.
-for y in ${REPO}/examples/contour/*.yaml ; do
-  ${KUBECTL} apply -f <(sed 's/imagePullPolicy: Always/imagePullPolicy: IfNotPresent/g' < "$y")
-done
+${KUBECTL} apply -f <(sed 's/imagePullPolicy: Always/imagePullPolicy: IfNotPresent/g' < ${REPO}/examples/contour/02-job-certgen.yaml)
+${KUBECTL} apply -f <(sed 's/imagePullPolicy: Always/imagePullPolicy: IfNotPresent/g' < ${REPO}/examples/contour/03-contour.yaml)
+${KUBECTL} apply -f <(sed 's/imagePullPolicy: Always/imagePullPolicy: IfNotPresent/g' < ${REPO}/examples/contour/03-envoy.yaml)
+
+${KUBECTL} apply -f - <<EOF
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: contour
+  namespace: projectcontour
+data:
+  contour.yaml: |
+    gateway:
+      name: contour
+      namespace: projectcontour
+    rateLimitService:
+      extensionService: projectcontour/ratelimit
+      domain: contour
+      failOpen: false
+    # The options below are copied from
+    # install-fallback-certificate.sh.
+    tls:
+      fallback-certificate:
+        name: fallback-cert
+        namespace: projectcontour
+    accesslog-format: envoy
+    disablePermitInsecure: false
```

Comment:
I believe these two settings are both defaults, so can be removed, but double-check me there.

Reply:
yup, they are the same.